addons/website_slides/static/src/js/slides_course_quiz.js

```diff
@@ -767,9 +769,9 @@ odoo.define('website_slides.quiz', function (require) {
         },
     });
 
-    return {
-        Quiz: Quiz,
-        ConfirmationDialog: ConfirmationDialog,
-        websiteSlidesQuizNoFullscreen: publicWidget.registry.websiteSlidesQuizNoFullscreen
+    export {
+        Quiz,
+        ConfirmationDialog,
     };
-});
+
+    export const websiteSlidesQuizNoFullscree = publicWidget.registry.websiteSlidesQuizNoFullscreen;
```

Comment:
This is no longer exported as default. If that's not a problem because you've adapted the imports, either do all exports inline (ie `export var Quiz = publicWidget...` higher in the file), or export a single object:

```js
    export {
        Quiz,
        ConfirmationDialog,
        publicWidget.registry.websiteSlidesQuizNoFullscreen as websiteSlidesQuizNoFullscree,
    };
```